app/controllers/api/integrations_controller.rb

```diff
@@ -228,14 +230,31 @@ def proxy_rules_attributes
   def proxy_params
     basic_fields = [
       :lock_version,
-
-      :auth_app_id, :auth_app_key, :api_backend, :hostname_rewrite, :oauth_login_url,
-      :secret_token, :credentials_location, :auth_user_key, :error_status_auth_failed,
-      :error_headers_auth_failed, :error_auth_failed, :error_status_auth_missing,
-      :error_headers_auth_missing, :error_auth_missing, :error_status_no_match,
-      :error_headers_no_match, :error_no_match, :error_status_limits_exceeded, :error_headers_limits_exceeded, :error_limits_exceeded,
-      :api_test_path, :policies_config, proxy_rules_attributes: %i[_destroy id http_method pattern delta metric_id
-                                                                   redirect_url position last], oidc_configuration_attributes: OIDCConfiguration::Config::ATTRIBUTES + [:id]
+      :auth_app_id,
+      :auth_app_key,
+      :api_backend,
+      :hostname_rewrite,
+      :oauth_login_url,
+      :secret_token,
+      :credentials_location,
+      :auth_user_key,
+      :error_status_auth_failed,
+      :error_headers_auth_failed,
+      :error_auth_failed,
+      :error_status_auth_missing,
+      :error_headers_auth_missing,
+      :error_auth_missing,
+      :error_status_no_match,
+      :error_headers_no_match,
+      :error_no_match,
+      :error_status_limits_exceeded,
+      :error_headers_limits_exceeded,
+      :error_limits_exceeded,
+      :api_test_path,
+      :policies_config,
+      proxy_rules_attributes: %i[_destroy id http_method pattern delta metric_id redirect_url position last],
+      oidc_configuration_attributes: OIDCConfiguration::Config::ATTRIBUTES + [:id],
```

Comment:
rebase from master plz 🙈

Reply:
It is up to date with master, what's wrong with this? 🤔

Reply:
right, my bad 👍🏼